src/Symfony/Component/Routing/Tests/Loader/AnnotationClassLoaderTest.php

```diff
@@ -107,6 +117,18 @@ public function testInvokableControllerLoader()
         $this->assertEquals(['https'], $routes->get('lol')->getSchemes());
     }
 
+    /**
+     * @requires PHP 8
+     */
+    public function testInvokableControllerWithAttributes()
+    {
+        $routes = $this->loader->load(\Symfony\Component\Routing\Tests\Fixtures\AttributeFixtures\InvokableController::class);
+        $this->assertCount(1, $routes);
+        $this->assertEquals('/here', $routes->get('lol')->getPath());
+        $this->assertEquals(['GET', 'POST'], $routes->get('lol')->getMethods());
+        $this->assertEquals(['https'], $routes->get('lol')->getSchemes());
```

Comment:
assertSame everywhere :)

Reply:
This should be fixed on the original test case then. This is merely C&P for now.

My plan is duplicate the fixtures and rewrite the annotations to attributes, so we can run the same test against both fixture sets.